frontend/app/src/comps/Positions/PositionCardLeverage.tsx

```diff
@@ -65,7 +65,7 @@ export function PositionCardLeverage({
               color: "positionContent",
             })}
           >
-            <div>Leverage loan</div>
+            <div>Multiply</div>
```

Comment:
Do we also call the object “Multiply”? For, we were having the app feature called “Leverage”, and the objects created from it “Leverage loan”.